circle.yml

```diff
@@ -0,0 +1,11 @@
+machine:
+  python:
+    version: 3.5.0
```

Comment:
Can't you just tell 3.5 instead of 3.5.0 (we want it to work with the latest version).
Eventually also 3.6